app/src/components/WalletSettings/ContactList/ContactList.vue

```diff
@@ -102,18 +102,25 @@ export default {
       newContact: '',
       newContactName: '',
       rules: {
-        required: value => !!value || 'Required'
+        required: value => !!value || this.t('walletSettings.required')
       },
-      verifierOptions: ALLOWED_VERIFIERS,
       ETH,
       saveContactAlert: false,
       saveContactAlertText: '',
       saveContactAlertType: 'success'
     }
   },
   computed: {
+    verifierOptions() {
+      const verifiers = JSON.parse(JSON.stringify(ALLOWED_VERIFIERS))
```

Comment:
Same as below

Reply:
This is for deep clone.. Cause i need to modify verifier labels with translation without affecting enums value.